app/src/test/java/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/util/VerificationCodeItemExtensionsTest.kt

```diff
@@ -36,10 +38,97 @@ class VerificationCodeItemExtensionsTest {
             authCode = nonFavoriteItem.code,
             favorite = (nonFavoriteItem.source as AuthenticatorItem.Source.Local).isFavorite,
             allowLongPressActions = true,
+            showMoveToBitwarden = false,
         )
 
-        assertEquals(expectedFavoriteItem, favoriteItem.toDisplayItem(alertThresholdSeconds))
-        assertEquals(expectedNonFavoriteItem, nonFavoriteItem.toDisplayItem(alertThresholdSeconds))
+        assertEquals(
+            expectedFavoriteItem,
+            favoriteItem.toDisplayItem(
+                alertThresholdSeconds = alertThresholdSeconds,
+                sharedVerificationCodesState = SharedVerificationCodesState.Error,
+            ),
+        )
+        assertEquals(
+            expectedNonFavoriteItem,
+            nonFavoriteItem.toDisplayItem(
+                alertThresholdSeconds = alertThresholdSeconds,
+                sharedVerificationCodesState = SharedVerificationCodesState.Error,
+            ),
+        )
+    }
+
+    @Test
+    @Suppress("MaxLineLength")
+    fun `toDisplayItem should only showMoveToBitwarden when SharedVerificationCodesState is Error`() {
```

Comment:
Should this be `...is Success`?

Reply:
Indeed it should! Thank you.

3a1d78c1359f8854abc11aa6e9472e4b6b599862